src/main/java/DukeException.java

```diff
@@ -0,0 +1,111 @@
+
+
+public class DukeException {
+    protected boolean isInvalidCommand;
+    protected boolean isNotDone;
+    protected boolean isDoneNoNumber;
+    protected boolean isTodoEmpty;
+    protected boolean isDeadlineEmpty;
+    protected boolean isEventEmpty;
+    protected boolean isDeadlineNoDate;
+    protected boolean isEventNoDate;
+    protected boolean isNotDelete;
+    protected boolean isDeleteNoNumber;
+    protected String Description;
```

Comment:
Consider using camel case for variable names as per coding standards. e.g "Description" can be renamed to "description".